extensions/wikia/SiteWideMessages/SpecialSiteWideMessages.php

```diff
@@ -133,32 +133,30 @@ function SiteWideMessagesSiteNoticeAfter( &$siteNotice ) {
  *
  * @author macbre
  */
-function SiteWideMessagesAddNotifications( Skin $skin, &$tpl ) {
-	global $wgExtensionsPath;
+function SiteWideMessagesAddNotifications(&$skim, &$tpl) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove the unused function parameters "$skim, $tpl". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1172)